docs/configuring-socorro.rst

```diff
@@ -8,6 +8,19 @@ Socorro produces graphs and reports, most are updated once per day.
 You must enter information about your releases into Socorro in order
 for this to work, and this information must match the incoming crashes.
 
+Configure Elasticsearch
+-----------------------
+
+Many features of Socorro depend on Elasticsearch, and to work you will first
```

Comment:
That's a valid point.  I guess it was the name of the script combined with the wording of both blocks that wasn't clear to me.  Perhaps something like:
> Supersearch, which is one of the most useful features of Socorro, depends on Elasticsearch.  The service needs to be initialised before it can be used:
[code-block]
For more comprehensive [etc...]